packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -731,10 +712,7 @@ export abstract class SharedObject<
 
 		let gcData: IGarbageCollectionData;
 		try {
-			const serializer = new SummarySerializer(
-				this.runtime.channelsRoutingContext,
-				(handle: IFluidHandle) => this.handleDecoded(handle),
```

Comment:
So during Summarization/GC we would have had a ton of "addedOutboundReference" calls... I wonder if that was providing some coverage that we're now missing, so we're seeing gcUnknownOutboundReferences.  Just a shot in the dark.

Reply:
Could be. But I would say that the system is more correct now than it was with this change.